### PR TITLE
Adding [SMART] SMART-NODE stake pool

### PR DESCRIPTION
**Pull request by CharlesAUGU**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -14,6 +14,21 @@
...
+ "social": {                                                           
+  "facebook": "", 
+  "twitter": "smartnodeFR",
+   "website": "https://smart-node.com/",  
+   "reddit": "",  
+   "instagram": "",
+   "youtube": ""
+ },
+ "telegram": "joinchat/AAAAAFWkMIPwtU4-FCOkeA",                                            
+ "operatorName": "SMART-NODE",
```
Comment (leo42) [inline registry.json:29]: Please use a real name or leave empty!

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (CharlesAUGU) [reply registry.json:29]: hello @leo42
I have made the change

Comment (leo42) [reply registry.json:29]: I was talking about the Operator name...

Telegram needs to be a personal telegram handle(so we can validate votes)

Comment (lophre) [reply registry.json]: You can leave the operatorName empty but you need to provide a valid private Telegram handle.

Comment (CharlesAUGU) [reply registry.json]: thank you @lophre 
I think it's ok now...

**Review by lophre — APPROVED** [review]

**Pull request merged**